### PR TITLE
📝 docs: Fix typo in Wilson's Twitter link

### PR DESCRIPTION
**Pull request by duckception**
🫡

### CODE REVIEW

**Review by wilsoncusack — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ opStackL1PublicClient.getOutputForL2Block(blockNumber: 2725977n, l2Chain: base)
 
 Check out the following places for more viem-related content:
 
-- Follow [@wilsoncuask](https://twitter.com/wilsoncuask) Twitter for project updates
+- Follow [@wilsoncuask](https://twitter.com/wilsoncusack) Twitter for project updates
```
Comment (wilsoncusack) [inline README.md:55]: err might want to fix both :) 
```suggestion
- Follow [@wilsoncusack](https://twitter.com/wilsoncusack) Twitter for project updates
```

**Review by wilsoncusack — CHANGES_REQUESTED** [review]

**Review by wilsoncusack — DISMISSED** [review]

**Comment by wilsoncusack** [comment]
there was a flaky test you may need to rebase sry :/

**Comment by duckception** [comment]
> there was a flaky test you may need to rebase sry :/

Done 🫡

File: README.md (hunk 1 above)
Comment (wilsoncusack) [reply README.md:55]: @duckception looks like this got reverted?

Comment (wilsoncusack) [inline README.md:55]:
```suggestion
- Follow [@wilsoncusack](https://twitter.com/wilsoncusack) Twitter for project updates
```

**Pull request merged**